bin/prover-client/src/operators/checkpoint.rs

```diff
@@ -62,7 +62,7 @@ impl CheckpointOperator {
     /// Retrieves the latest checkpoint index
     pub async fn fetch_latest_ckp_idx(&self) -> Result<u64, ProvingTaskError> {
         self.cl_client
-            .get_latest_checkpoint_index()
+            .get_latest_checkpoint_index(None)
```

Comment:
I thought we realized this was only populated when we're running a sequencer and we should use some other construct?  Or is this safe here?

Reply:
@sapinb might have some inputs on this.